examples/calibration/plot_calibration_curve.py

```diff
@@ -80,7 +80,7 @@ def plot_calibration_curve(est, name, fig_index):
     # Logistic regression with no calibration as baseline
     lr = LogisticRegression(C=1.)
 
-    fig = plt.figure(fig_index, figsize=(10, 10))
+    plt.figure(fig_index, figsize=(10, 10))
     ax1 = plt.subplot2grid((3, 1), (0, 0), rowspan=2)
     ax2 = plt.subplot2grid((3, 1), (2, 0))
```

Comment:
Hmm, so we can't do `fig.subplot2grid` ?

Reply:
what's wrong with the current code?  If `fig` was not used, there is no point assigning this variable.

Reply:
I do not think `fig.subplot2grid` exists. I added `plt.subplot2grid(..., fig=fig)` to be more OO.